internal/cmd/flink/command_compute_pool_describe.go

```diff
@@ -0,0 +1,55 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newComputePoolDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "describe [id]",
```

Comment:
[id] and <id>?

Reply:
This should be `[id]` because it's optional.